entity-framework/core/extensions/index.md

```diff
@@ -54,6 +54,15 @@ Entity Framework Core Generator (efg) is a .NET Core CLI tool that can generate
 
 ## Extensions
 
+### Microsoft.EntityFrameworkCore.DistributedCache
+An extension for caching result of queries into redis cache.
```

Comment:
Please fix the broken English (or let me know if you need help with that)

Reply:
Hello. I did. Please correct me if I did any mistake. Thanks.